fastify.js

```diff
@@ -358,6 +360,16 @@ function build (options) {
     return this._contentTypeParser.hasParser(contentType)
   }
 
+  function handleClientError (e, socket) {
+    socket.end(
+      JSON.stringify({
```

Comment:
This can be expensive: could it be built on the fastify start up keeping the string in memory?